pkg/server/plugin/nodeattestor/base/base.go

```diff
@@ -22,6 +26,15 @@ func (p *Base) BrokerHostServices(broker pluginsdk.ServiceBroker) error {
 	return nil
 }
 
-func (p *Base) IsAttested(ctx context.Context, agentID string) (bool, error) {
-	return agentstore.IsAttested(ctx, p.store, agentID)
+func (p *Base) AssessTOFU(ctx context.Context, agentID string, log hclog.Logger) error {
```

Comment:
can you add an unit test to validate this new feature?

Reply:
Yup. Added.